backend/src/main/java/ai/verta/modeldb/authservice/AuthServiceChannel.java

```diff
@@ -178,6 +180,22 @@ private void initOrganizationServiceStubChannel() {
     return organizationServiceBlockingStub;
   }
 
+  private void initAuditLogServiceStubChannel() {
+    Metadata requestHeaders = getMetadataHeaders();
+    LOGGER.trace("Header attaching with stub : {}", requestHeaders);
+    ClientInterceptor clientInterceptor = MetadataUtils.newAttachHeadersInterceptor(requestHeaders);
+    auditLogServiceBlockingStub =
+        AuditLogServiceGrpc.newBlockingStub(authServiceChannel).withInterceptors(clientInterceptor);
+    LOGGER.trace("Header attached with stub");
+  }
+
+  public AuditLogServiceGrpc.AuditLogServiceBlockingStub getAuditLogServiceBlockingStub() {
+    if (auditLogServiceBlockingStub == null) {
+      initAuditLogServiceStubChannel();
+    }
+    return auditLogServiceBlockingStub;
+  }
+
```

Comment:
@ad-47 Can you add a PR comment explaining this change please?

Reply:
we have created AuthServiceChannel.java for getting grpc client stub for the UAC microservices. For each microservice like Auth service, Role Service, Collaborator service we have created `newBlockingStub(authServiceChannel)` so here as same way I have created the same for Audit service.

Reply:
👍